language/libs/oslib.rd

```diff
@@ -108,12 +109,16 @@ items in linux {
 }
 
 items in redhat_family {
-  @cfengine_name="(redhat.!centos.!fedora.!oracle_linux.!scientific_linux.!amazon_linux)"
+  # Ugly but needed as very old rhel do not have the rhel class
+  @cfengine_name="rhel|(redhat.!centos.!fedora.!oracle.!almalinux.!rocky.!scientific_linux.!amazon_linux)"
   rhel,
   centos,
   fedora,
-  oracle_linux,
+  oracle,
```

Comment:
oracle_linux doesn't exist in cfengine ?

Reply:
not on a Oracle Linux 7 at east:

```
rudder agent info -v | grep -E "(redhat|rhel|centos|oracle)"
  oracle
  oracle_7
  oracle_7_7
  redhat
  redhat_7
  redhat_7_7
  redhat_s
  redhat_s_7
  redhat_s_7_7
```

Reply:
I don't know where `oracle_linux` comes from, maybe older agents and/or older Oracle Linux versions